shared/httputil/httputil.go

```diff
@@ -40,14 +40,17 @@ func IsHTTPS(r *http.Request) bool {
 
 // SetCookie writes the cookie value.
 func SetCookie(w http.ResponseWriter, r *http.Request, name, value string) {
+	// the cookie value (token) is responsible for expiration
+	cookieMaxAgeSeconds := 2147483647
```

Comment:
Why was `2147483647` seconds chosen here?

Reply:
Ah, I guess we want to "never expire" here and `2147483647` is the 32-bit integer limit.